etcd/fs/fs_test.go

```diff
@@ -57,23 +92,14 @@ func TestFs1(t *testing.T) {
 	}
 	//var etcdFs afero.Fs = NewEtcdFs()
 
-	if err := etcdFs.Mkdir("/", umask); err != nil {
```

Comment:
I nuked:

```
if err := etcdFs.Mkdir("/", umask); err != nil {
	t.Logf("error: %+v", err)
	if err != etcdfs.ErrExist {
		return
	}
}
```
because it makes no sense to `mkdir /` and always results in a File Exists error.

I got rid of FsTest2 because all it did was check if `/` and `/tmp` existed, but it never made `/tmp` during the test, so would always fail with a clean etcd node.